android/src/main/java/com/pushwithgcm/PushWithGCMListenerService.java

```diff
@@ -0,0 +1,6 @@
+package com.pushwithgcm;
+
+import com.google.android.gms.gcm.GcmListenerService;
+
+public class PushWithGCMListenerService extends GcmListenerService {
```

Comment:
OK. Is it needed in this package, or should the app using it create the listener? 
Our app use GCM Messages of type "notification", but we can't say that for other apps that use this package.